### PR TITLE
 Added token binding detailed documentation

### PR DESCRIPTION
**Pull request by himeshsiriwardana**
## Purpose

screencapture-localhost-8000-asgardeo-docs-references-token-binding-client-request-2024-02-15-18_17_03

Related issue: https://github.com/wso2/product-is/issues/18804

### CODE REVIEW

File: en/includes/references/token-binding.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+    - If the request contains a `tokenBindingId`, {{product_name}} issues a new access token for each unique `tokenBindingId` value. Issuing a new token does not revoke tokens issued for other `tokenBindingId` values.
+
+- **Opaque**
+
+    - If the request does not contain a `tokenBindingId`, {{product_name}} issues the same access token for every new token request until it expires.
+
+    - If the request contains a `tokenBindingId`, {{product_name}} issues a new access token for each unique `tokenBindingId` value. Issuing a new token does not revoke tokens issued for other `tokenBindingId` values.
+
+### Try it out
```
Comment (VivekVinushanth) [inline en/includes/references/token-binding.md:52]: Will it be clumsy if we add try it out for all the binding types in the same doc?

Comment (VivekVinushanth) [reply en/includes/references/token-binding.md:52]: +1 to move this each binding type to new section once we have comprehensive doc for other types.

File: en/includes/references/token-binding.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    curl -X POST
+    -u "client_id:client_secret"
+    -H "Content-Type: application/x-www-form-urlencoded"
+    -d "token=<your_access_token>"
+    https://localhost:9443/oauth2/introspect
+    ```
+
+5. Repeat the steps for different tokenBindingId values and different token types and verify that the client-request behavior conforms to the patterns mentioned above.
+
+Eventhough the **client-request** token binding type is tailored towards back-channel grant typee, its versatility extends to support any other grants in {{product_name}}. This flexibility allows developers to incorporate this feature into various use-cases, providing a more robust and secure solution for identity and access management.
```
Comment (VivekVinushanth) [inline en/includes/references/token-binding.md:95]: typee => type

**Review by VivekVinushanth — DISMISSED** [review]

File: en/includes/references/token-binding.md (hunk 1 above)
Comment (himeshsiriwardana) [reply en/includes/references/token-binding.md:52]: We will separate the token types into their own docs when we have the content.

**Review by VivekVinushanth — APPROVED** [review]

**Pull request merged**